quick-start/src/main/java/com/marklogic/quickstart/model/SearchPathModel.java

```diff
@@ -22,8 +22,9 @@
 	public String absolutePath;
 
 	public SearchPathModel(String name, String relativePath, String absolutePath) {
+        this.name = name;
```

Comment:
I hate to nitpick... but are you using editorconfig to get your spacing right?

Reply:
No idea why this is appearing this way here. It looks correct in my editor. Editorconfig is installed and applied by default to the entire project, and the project's checked-in .editorconfig is available, which sets the indent on Java files to 4 spaces. Perhaps this is just a formatting issue with the pull request viewer.